pkg/cmd/clean/exec.go

```diff
@@ -79,7 +79,8 @@ func (o *Options) Run() error {
 		return nil
 	}
 	b := retry.DefaultBackoff
-	b.Duration = 1 * time.Second
+	b.Duration = 5 * time.Second
+	b.Steps = 15
```

Comment:
Why do we need larger retry and step after the json output change? Is this unrelated fix to avoid failures in the CI?

Reply:
yes, this is related to avoid failures in the CI.